domains/runtime/auto-id/build.rs

```diff
@@ -1,7 +1,7 @@
 fn main() {
     #[cfg(feature = "std")]
     {
-        std::env::set_var("WASM_BUILD_TYPE", "release");
+        unsafe { std::env::set_var("WASM_BUILD_TYPE", "release") };
```

Comment:
Please add a safety comment to each of these:
> SAFETY: safe to call in single threaded programs, build scripts are single threaded

Reply:
I'm not sure it is single-threaded, it is certainly not guaranteed. A better proof is to say that when this code runs, there is only a single thread active. It doesn't matter if there are more threads later. See https://doc.rust-lang.org/stable/std/env/fn.set_var.html#safety

> The exact requirement is: you must ensure that there are no other threads concurrently writing or _reading(!)_ the environment through functions or global variables other than the ones in this module.